receiver/jmxreceiver/testdata/integration/1.46.0-alpha-scraper-custom-jmxconfig/simple-cassandra.yaml

```diff
@@ -0,0 +1,9 @@
+rules:
+  # Only one Storage metric
+  - bean: org.apache.cassandra.metrics:type=Storage,name=TotalHints
```

Comment:
tomcat might be a simpler and faster choice for testing, as cassandra is quite heavy to run and tends to take a bit of time to start.

Reply:
Sounds like a good improvement to me. Opened this issue to tackle it in a follow-up PR: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/41103